packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeRenderer.tsx

```diff
@@ -28,7 +27,7 @@ interface TreeRendererOwnProps {
 /** @alpha */
 type TreeRendererProps = Pick<ReturnType<typeof useTree>, "expandNode"> &
   Partial<Pick<ReturnType<typeof useTree>, "selectNodes" | "isNodeSelected" | "getHierarchyLevelDetails" | "reloadTree">> &
-  Omit<TreeNodeRendererProps, "node" | "reloadTree"> &
+  Omit<TreeNodeRendererProps, "node" | "reloadTree" | "aria-level" | "aria-posinset" | "aria-setsize"> &
```

Comment:
I guess `aria` props are not needed.

Reply:
They are being omitted.

Reply:
Aren't they already emitted here? https://github.com/iTwin/presentation/blob/596bf2fac592f5da9c4ac896d74898aa8a5d3aa0/packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeNodeRenderer.tsx#L18

Also looking at that `TreeNodeProps` type. Can we move `Omit` parts to https://github.com/iTwin/presentation/blob/596bf2fac592f5da9c4ac896d74898aa8a5d3aa0/packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeNodeRenderer.tsx#L47 to avoid having multiple places omitting props from same base type.

Reply:
Yes, you are right, forgot that node renderer needed to omit those props also. Changed both types

Reply:
Can we move `Omit` part to the `TreeNodeRenderer` type definition and keep `type TreeNodeProps = ComponentPropsWithoutRef<typeof Tree.Item>`

Reply:
Moved it out